Sources/web3swift/Utils/EIP/EIP712.swift

```diff
@@ -38,7 +38,6 @@ public extension EIP712Hashable {
     }
 
     func hash() throws -> Data {
-        typealias SolidityValue = (value: Any, type: ABI.Element.ParameterType)
```

Comment:
This type alias wasn't used.

Reply:
Just wonder, how did you figured it out? I mean is there any interesting tool that you've using for that? This is more like off topic curiosity question rather then PR comment.

Reply:
Answering [your question](https://github.com/web3swift-team/web3swift/pull/725#discussion_r1069663928): I just had an issue in this function after on line 59. Then started looking at everything we have in the function to figure out what is happening and noticed that we have a `typealias` at the very top that isn't used anywhere in the function. 
Then removed it. No fancy tools, just plain old refactoring.